### ISSUE TITLE
Failing test `/tests/plugins/a11yhelp/manual/dialogfocusinfo`

### ISSUE THREAD

**Issue by sculpt0r** [ckeditor/ckeditor4#5276]
## Type of report

Failing test

## Provide detailed reproduction steps (if any)

1. Run test http://0.0.0.0:1030/tests/plugins/a11yhelp/manual/dialogfocusinfo
2. Perform step 1

### Expected result

Tester is able to perform step 2

### Actual result

*What is the actual result of the above steps?*

It is impossible to use `ALT+0` on mobile device.

## Other details

* Browser: Chrome mobile 103 (Android)
* OS: Android 11
* CKEditor version: 4.19.1 (pre-release)
* Installed CKEditor plugins: full all

**Comment by jacekbogdanski** [comment]
I see it was already corrected in https://github.com/ckeditor/ckeditor4/pull/5260
